database/S/Secret Sign/Aves Noir/Aves Noir

```diff
@@ -0,0 +1,55 @@
+Spread out? your wings
+... as there's night?
+
+The world down there fear you
```

Comment:
Sure this is "fear" and not "fears"?

Reply:
I can't hear "s", but it looks like the first part is wrong, the line is likely `They all down there fear you` 

Just seeing this, sorry for the late reply.

I'm going to give a couple more iterations for this branch, then try to get in touch with the author. I only managed to get to the guitarist last year, and he didn't write any of these lyrics.